rpc/wrtc_call_queue_mongodb.go

```diff
@@ -241,8 +260,13 @@ type mongoDBWebRTCCallQueue struct {
 	csStateUpdates              chan changeStreamStateUpdate
 	csCtxCancel                 func()
 
-	// function to update access times on robot parts based on this call queue
-	activeAnswerersfunc *func(hostnames []string, atTime time.Time)
+	// function passed in during construction to update last_online timestamps for documents
+	// in robot and robot_part based on this call-queue/operator.
+	onAnswererLiveness func(hostnames []string, atTime time.Time)
```

Comment:
Renamed `activeAnswererFunc` -> `onAnswererLiveness` for clarity/consistency with names in `app`. Changed from `*func` -> `func`, too, as I think we only had that to let `activeAnswerersFunc == nil`, but all `func`s can be `nil`.